### ISSUE TITLE
Webcam

### ISSUE THREAD

**Issue by wissehes** [Rambalac/GMaster#27]
Hello, can you use your lumix device as a webcam with this program?

**Comment by Rambalac** [comment]
No. But you would not use it as webcam as almost any webcam has much higher quality than image sent by any Lumix camera over WiFi.

**Comment by wissehes** [comment]
Yeah, but it still can be a handy feature

**Comment by kpym** [comment]
> No. But you would not use it as webcam as almost any webcam has much higher quality than image sent by any Lumix camera over WiFi.

The standrad webcams do not have zoom. The standard webcams have very big depth of field.
So IMO it will be nice if we can use this app to simulate a webcam.

Thanks for your work! 🙏
